contracts/network/election.go

```diff
@@ -0,0 +1,160 @@
+package network
+
+//go:generate solc --abi --bin --overwrite -o build contracts/Election.sol
+//go:generate abigen -abi build/Election.abi -bin build/Election.bin -pkg contracts -type ElectionContract -out contracts/election.go
```

Comment:
Shouldn't the output be `contracts/network/election.go`?

Reply:
I followed ethereum's way -This process generates the contract bindings for go. The bindings usually stay in the same package as the contract and the wrapper type stays in the network package.